core/src/main/java/org/apache/accumulo/fate/util/Retry.java

```diff
@@ -67,6 +67,16 @@ private Retry(long maxRetries, long startWait, long waitIncrement, long maxWait,
     this.logIntervalNanoSec = MILLISECONDS.toNanos(logInterval);
     this.hasNeverLogged = true;
     this.lastRetryLog = -1;
+    this.waitFactor = 1;
+  }
+
+  public long getWaitFactor() {
+    return waitFactor;
+  }
+
+  public void setWaitFactor(long waitFactor) {
```

Comment:
This should follow the same pattern as the rest of the options, by setting it in the constructor, and building it with the fluent RetryFactory.